nautobot/extras/templates/extras/dynamicgroup.html

```diff
@@ -33,8 +54,18 @@
             <pre>{{ object.filter | render_json }}</pre>
         </div>
     </div>
-{% endblock content_left_page %}
-
-{% block content_right_page %}
-    {% include 'utilities/obj_table.html' with table=members_table table_template='panel_table.html' heading='Dynamic Group members' %}
 {% endblock content_right_page %}
+
+{% block content_full_width_page %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Child Groups</strong>
+        </div>
+        <div class="panel-body">
+            {% include 'utilities/obj_table.html' with table=children_table table_template='panel_table.html' heading='Children' no_paginate=True %}
```

Comment:
🦤 - Rename `no_paginate=True` to something more intuitive like `disable_pagination=True`. This needs to be a positive boolean so that reads negatively since it needs to be backwards-compatible.